manifests/host.pp

```diff
@@ -11,7 +11,7 @@
 
   $host = $name
 
-  include dhcp::params
+  include ::dhcp::params
```

Comment:
could you test that? I thought that doesn't work on puppet3 with STRICT_VARIABLES enabled?